mesonbuild/modules/_qt.py

```diff
@@ -648,3 +733,431 @@ def compile_translations(self, state: 'ModuleState', args: T.Tuple, kwargs: 'Com
             return ModuleReturnValue(results.return_value[0], [results.new_objects, translations])
         else:
             return ModuleReturnValue(translations, [translations])
+
+    def _source_to_files(self, state: 'ModuleState', sources: T.Sequence[T.Union[FileOrString, build.GeneratedTypes]]) -> T.List[File]:
```

Comment:
Quoted annotations are no longer necessary since we consistently use `from __future__ import annotations` in all files.

Reply:
fixed, I also added a commit to update align the rest of the class on this policy

Reply:
Sure, I guess.

Both styles "work" but the newer style allows for shorter lines (by two columns per annotation). In general we haven't bothered to fix old code, but it's always best to have new code so things the preferred way. Some old code has been naturally rewritten over time for unrelated reasons, and inherited the new annotation style along the way, so we will eventually converge on a consistent codebase.

Fixing some of them immediately is fine (though I should rebase and update my PR that adds an ignore-revs file for `git blame`!) especially if you're going to be working on this file a lot and would find it helpful. :D

I just wouldn't ever demand to fix the whole file in a review request.